src/index.js

```diff
@@ -1,4 +1,48 @@
-import _ from 'lodash';
 import './style.css';
 
+const taskContainer = document.querySelector('.all-tasks');
+const tasks = [
+  {
+    id: Date.now(),
+    completed: false,
+    description: 'Make Breakfast',
+  },
+  {
+    id: Date.now(),
+    completed: false,
+    description: 'Clean the House',
+  },
+  {
+    id: Date.now(),
+    completed: false,
+    description: 'Report to Work',
+  },
+  {
+    id: Date.now(),
+    completed: false,
+    description: 'join stand up team',
+  },
```

Comment:
- [ ] As I mentioned previously each task object should have `description`, `completed` & am `index` property as this is part of the project requirement:
![image](https://github.com/coker2019/To-do-List/assets/104968236/4348c562-0666-4335-909d-3301c1f3eac2)